src/fides/api/ctl/migrations/versions/9f38dad37628_redo_roles_scopes_mapping.py

```diff
@@ -0,0 +1,59 @@
+"""Redo roles scopes mapping
+
+Revision ID: 9f38dad37628
+Revises: eb1e6ec39b83
+Create Date: 2023-03-03 16:53:03.553992
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+from sqlalchemy import text
+from sqlalchemy.engine import ResultProxy
+
+revision = "9f38dad37628"
+down_revision = "eb1e6ec39b83"
+branch_labels = None
+depends_on = None
+
+
+def swap_roles(old_role: str, new_role: str):
+    bind = op.get_bind()
+    matched_rows: ResultProxy = bind.execute(
+        text("SELECT * FROM fidesuserpermissions WHERE :old_role = ANY(roles);"),
+        {"old_role": old_role},
+    )
+    for row in matched_rows:
+        current_roles = row["roles"]
+        current_roles.remove(old_role)
+        current_roles.append(new_role)
+        bind.execute(
```

Comment:
you're a hero for doing this—I probably would've been too lazy 😅

Reply:
ha I doubt anyone will run into this, I didn't know if we'd get them out in the same release